lib/better_words_web/router.ex

```diff
@@ -18,6 +18,13 @@ defmodule BetterWordsWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+
+    live "/worst_words", WorstWordLive.Index, :index
+    live "/worst_words/new", WorstWordLive.Index, :new
+    live "/worst_words/:id/edit", WorstWordLive.Index, :edit
+
+    live "/worst_words/:id", WorstWordLive.Show, :show
+    live "/worst_words/:id/show/edit", WorstWordLive.Show, :edit
```

Comment:
These routes were added by hand, but the actual code was output from running the generator with directions on where to paste it.